exercises/ruby-1-a/ruby_1_a.rb

```diff
@@ -1,3 +1,13 @@
-def word_with_most_vowels(sentence)
-  # Start your code here.
+class ShoppingCart
+  def initialize(items)
+    fail NotImplementedError.new('Please implement the ShoppingCart initializer')
```

Comment:
I'm not that well versed in Ruby, but is "initializer" something people will be familiar with? Instead of doing:

```suggestion
    fail NotImplementedError.new('Please implement the ShoppingCart#initialize method')
```

Reply:
I like the `NotImplementedError`!

Reply:
In Ruby when you call `Class.new`, Ruby calls the `initialize` method. The method new will create a new instance of the class, and call initialize on it.

That's the reason why I wasn't sure about `ShoppingCart#initialize` or `ShoppingCart#new` but I guess is fine to go with `ShoppingCart#initialize`

Reply:
Thanks for explaining.